lib/ui/widgets/pages/statistic_page.dart

```diff
@@ -12,28 +16,39 @@ class StatisticPage extends StatelessWidget {
   final ValueChanged<String> onItemDelete;
 
   static const _icon = Icon(Icons.sms_failed_outlined);
+  static const _deleteIcon = Icon(Icons.delete, color: Palette.white);
 
   @override
   Widget build(BuildContext context) => AppWrapper(
         child: ListView.builder(
           itemCount: failedQuestions.length,
           itemBuilder: (context, index) {
+            final question = failedQuestions[index];
             return Dismissible(
-              key: ValueKey(failedQuestions[index]),
+              key: ValueKey(question),
```

Comment:
I dont this what we need ValueKey, try to delete and check how it work

Reply:
parameter key is required